skeleton/webpack/webpack.config.js

```diff
@@ -21,32 +21,41 @@ const testDir = path.resolve(__dirname, 'test', 'unit');
 const nodeModulesDir = path.resolve(__dirname, 'node_modules');
 const baseUrl = '/';
 
-const cssRules = [
-  { loader: 'css-loader' },
-  // @if feat['postcss-basic']
-  {
-    loader: 'postcss-loader',
-    options: { plugins: () => [require('autoprefixer')()] }
-  }
-  // @endif
-  // @if feat['postcss-typical']
-  {
-    loader: 'postcss-loader',
-    options: { plugins: () => [
-      require('autoprefixer')(),
-      require('cssnano')()
-    ] }
-  }
-  // @endif
-];
+const cssRules = (production) => {
+  return [
+    { loader: 'css-loader' },
+    // @if feat['postcss-basic'] || feat['postcss-typical'] || feat['postcss-tailwind']
+    {
+      loader: 'postcss-loader',
+      options: {
+        plugins: () => [
+          require('autoprefixer')(),
+          // @if feat['postcss-tailwind']   
+          require('tailwindcss')('tailwind.config.js'),
```

Comment:
Just a question, why we need a tailwind.config.js? I see it works without it on tailwind webpack doc.

Reply:
1. I have added this for ease of use. Especially for novices.
2. A very common example for me is RTL. You should add a variant like the following to support RTL.
```js
module.exports = {
  theme: {
    extend: {}
  },
  variants: {
	textAlign: ['responsive', 'rtl']
  },
  plugins: [
	function ({ addVariant }) {
		addVariant('rtl', ({ modifySelectors, separator }) => {
			modifySelectors(({ className }) => {
				return `[dir="rtl"] .rtl\\${separator}${className}`
			})
		})
	}
  ]
}
```

I always need such a code.

Reply:
Do you agree with me to keep `tailwind.config.js`?

Reply:
Yes, I was only raising a question, not a request.